ORIGIN_TRIAL_FEEDBACK.md

```diff
@@ -0,0 +1,22 @@
+# Origin Trial Feedback
+
+1. The API was easy to use.
+2. The API would be a lot more useful if it was available in the background (i.e. from push). See #5, #28
+
+## Other Requests
+Have the API work on tabs.
+- This would not change the API design.
+- We would like to add this in future.
+
+
+Make it possible to change the Badge colour.
+- Platform support would be limited (possibly only Windows).
+- Could be explored as a future extension.
+
+## Implementation Specific Feedback (Chrome)
```

Comment:
I think we can probably remove this from here since it's implementation specific